src/pybind/mgr/dashboard/conftest.py

```diff
@@ -0,0 +1,26 @@
+import sys
```

Comment:
I left that file there because that's the primary location `pytest` looks for, and as this (does/could) also affect to Doctests (we might `mock` the RBD ones there) or other non-`test/` files (e.g.: `ci/`). It's basically initialization/mocking logic common to all testing stuff. Nevertheless I have no problem if someone wants to move this into `test/`.